enterprise/models.py

```diff
@@ -1787,3 +1788,22 @@ def clean(self):
 
         if validation_errors:
             raise ValidationError(validation_errors)
+
+
+class SystemWideEnterpriseRole(UserRole):
+    """
+    System wide user role definitions specific to Enterprise.
+    """
+    pass
+
+
+class SystemWideEnterpriseUserRoleAssignment(UserRoleAssignment):
+    """
+    Model to map users to a SystemWideEnterpriseRole.
+    """
+    role_class = SystemWideEnterpriseRole
+
+    @property
+    def enterprise_customer_uuid(self):
+        enterprise_user = EnterpriseCustomerUser.objects.get(user=self.user)
```

Comment:
@brittneyexline I think this query needs to be changed to `user_ud=self.user.id`